backend/pkg/elasticsearch/bootstrapper/count.go

```diff
@@ -9,6 +9,9 @@ var countIndex = map[string]interface{}{
 	},
 	"mappings": map[string]interface{}{
 		"properties": map[string]interface{}{
+			"created_at": map[string]interface{}{
```

Comment:
Added created_at for the search_after query. This will be instrumental in our future pipeline efforts